### PR TITLE
Update database using GitHub Actions

### PR DESCRIPTION
**Pull request by abarichello**

### CODE REVIEW

**Comment by abarichello** [comment]
Falta terminar [esse daqui](https://github.com/caravelahc/moita/pull/1) pro request funcionar com CORS, mas dá uma olhada aí

File: .gitignore [hunk 1]
```diff
@@ -21,10 +18,3 @@ capim.py
 .htaccess
 dispatch.cgi
 dispatch.fcgi
-
-#-------------------------------------------------------------------------------
-# Python
-
-.venv
-*.pyc
-__pycache__
```
Comment (JPTIZ) [inline .gitignore]: We still have python code (under `db/src`), why removing these then?

File: html/capim.html [hunk 2]
```diff
@@ -129,22 +106,12 @@
...
-            </div>
-        </div>
-
         <div id="dconsole" style="font-size:11px;">
         </div>
 
         <div id="grayout" style="display:none;"></div>
         <div id="sobre_popup" style="display:none;">
-            include_sobre
+            <a>Sobre</a>
```
Comment (JPTIZ) [inline html/capim.html:110]: I'm not sure we should replace with this. Originally, [the Makefile rule for index.html](https://github.com/caravelahc/capim/blob/master/Makefile#L56) would replace this with the entire contents of [sobre.html](https://github.com/caravelahc/capim/blob/master/html/sobre.html), which I think that has some good info that need to be kept (about authorship, usage, maintance and privacy). If you want to replace, I think you should do it with the entire contents of `sobre.html` as expected or at least cover the info that was there previously.

File: js/main.js [hunk 3]
```diff
@@ -860,10 +850,8 @@ function Main(ui_materias, ui_turmas, ui_logger, ui_combinacoes, ui_horario,
         if (semestre == current) {
             ui_avisos.reset();
         } else {
-            let str = semestre.substr(0,4) + "-" + semestre.substr(4,1);
             ui_avisos.set_text(
-                "Você escolheu os horários de " + str + "! " +
-                "Nós já estamos em " + year + '-' + semester + "!"
+                "Há semestres novos disponíveis"
```
Comment (JPTIZ) [inline js/main.js:848]: :+1:

File: .gitignore (hunk 1 above)
Comment (abarichello) [reply .gitignore]: Vo bota depois

File: html/capim.html (hunk 2 above)
Comment (abarichello) [reply html/capim.html:110]: Já tem tudo isso nesse repo que ja tem um link no matrufsc, melhor do que editar o popup zoadasso

Comment (JPTIZ) [reply html/capim.html:110]: Acho que seria bom então linkar para o Readme do repo, o que acha?

Comment (abarichello) [reply html/capim.html:110]: Mas o link pro repo já mostra o README na página principal 🤔

**Review by JPTIZ — APPROVED** [review]
LGTM

Bora ver isso quebrar

**Pull request merged**